aws-ssm-document/src/main/java/com/amazonaws/ssm/document/DeleteHandler.java

```diff
@@ -53,22 +53,21 @@
         final CallbackContext callbackContext,
         final Logger logger) {
 
+        final CallbackContext context = callbackContext == null ? CallbackContext.builder().build() : callbackContext;
```

Comment:
can we define the overriden CallbackContext in the base method in the parent class?

Reply:
BaseHandler is a generated class from framework, any modifications to it will be gone when the package is built.